themes/vscode.toml

```diff
@@ -0,0 +1,28 @@
+# This theme is derived from the color scheme in
+# https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts
+# Original code is licensed under the MIT License.
+# Copyright (c) Microsoft Corporation.
```

Comment:
Can't really license colors under a source code license, it doesn't make sense. And no source code of Microsoft is part of this file.

So I think it's best to remove this.